docs/src/main/sphinx/sql/create-table-as.md

```diff
@@ -15,9 +15,14 @@ AS query
 Create a new table containing the result of a {doc}`select` query.
 Use {doc}`create-table` to create an empty table.
 
+The optional `OR REPLACE` clause replaces the existing table
```

Comment:
The optional `OR REPLACE` clause causes an existing table with the specified name to be replace with the new table definition. Support for table replacement varies across connectors. Refer to the connector documentation for details.